### PR TITLE
Add Release-Notes for 1.32.1

### PR DESCRIPTION
**Pull request by tippexs**

### CODE REVIEW

**Review by tarynmusgrave — APPROVED** [review]
This looks good to me.

**Comment by Jcahilltorre** [comment]
Nit for future releases: the branch name should be `release-1.32.1` (to be able to protect all the branches that match the release-xxx pattern)

File: source/news/2024/index.rst [hunk 1]
```diff
@@ -4,6 +4,14 @@ News of 2024
 
 News archive for the year 2024.
 
+.. nxt_news_entry::
+   :author: Unit Team
+   :description: Version 1.32.1 is a maintenance release, fixing bugs in the new WebAssembly Language Module and in our NJS implementation.
```
Comment (Jcahilltorre) [inline source/news/2024/index.rst:9]:
```suggestion
   :description: Version 1.32.1 is a maintenance release that fixes bugs in the new WebAssembly Language Module and in our NJS implementation.
```

File: source/news/2024/unit-1.32.1-released.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
+:orphan:
+
+####################
+Unit 1.32.1 Released
+####################
+
+NGINX Unit 1.32.1 is a maintenance release, fixing bugs in the new WebAssembly Language Module and in our NJS implementation.
```
Comment (Jcahilltorre) [inline source/news/2024/unit-1.32.1-released.rst:7]:
```suggestion
NGINX Unit 1.32.1 is a maintenance release that fixes bugs in the new WebAssembly Language Module and in our NJS implementation.
```

File: source/news/2024/unit-1.32.1-released.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+This release fixes the following issues:
+
+**************************************************************
+Applications of type `wasm-wasi-component` can't be restarted
+**************************************************************
+
+Applications deployed as `wasm-wasi-components` can't be restarted using the `restart` endpoint.
+
+After deploying a new Wasm Component binary to disk the restart will trigger a reload of the component in Unit without restarting the server.
```
Comment (Jcahilltorre) [inline source/news/2024/unit-1.32.1-released.rst:21]:
```suggestion
After deploying a new Wasm Component binary to disk, the restart will trigger a reload of the component in Unit without restarting the server.
```

File: source/news/2024/unit-1.32.1-released.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+As restarts will work independently of the application type, the behavior shipped with **1.32.0** was not right and fixed with this release.
```
Comment (Jcahilltorre) [inline source/news/2024/unit-1.32.1-released.rst:23]:
```suggestion
As restarts will work independently of the application type, the behavior shipped with **1.32.0** was not right. It has been fixed in this release.
```

File: source/news/2024/unit-1.32.1-released.rst [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+
+************************************************************
+Unit-variables in NGINX JavaScript are constantly cached
+************************************************************
+
+With **1.32.0** we have released the possibility to access all Unit variables form inside NJS.
```
Comment (Jcahilltorre) [inline source/news/2024/unit-1.32.1-released.rst:30]:
```suggestion
In **1.32.0** we added the possibility to access all Unit variables form inside NJS.
```

File: source/news/2024/unit-1.32.1-released.rst [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+As reported in GitHub issue `#1169 <https://github.com/nginx/unit/issues/1169>`__ the variables are cached and will hold the wrong value, which is definitely not the way this feature should work. With 1.32.1 we have fixed this issue.
```
Comment (Jcahilltorre) [inline source/news/2024/unit-1.32.1-released.rst:32]:
```suggestion
As reported in GitHub issue `#1169 <https://github.com/nginx/unit/issues/1169>`__ the variables were cached and would hold the wrong value, which is not how this feature should work. With version 1.32.1, we have fixed this issue.
```

**Review by Jcahilltorre — APPROVED** [review]

**Review by danielledeleo — CHANGES_REQUESTED** [review]
source/conf.py:8
```
version = '1.32.0'
release_date = 'Feb 27, 2024'
```

These need to be bumped before release if not in this PR, then in another.

**Comment by Jcahilltorre** [comment]
> source/conf.py:8
> 
> ```
> version = '1.32.0'
> release_date = 'Feb 27, 2024'
> ```
> 
> These need to be bumped before release if not in this PR, then in another.

I have bumped them, please remove the change request.

**Review by danielledeleo — APPROVED** [review]

**Pull request merged**